src/debug/ee/debugger.cpp

```diff
@@ -13938,25 +13911,23 @@ void GenericHijackFuncHelper()
     // Get the continue type. Non-zero means that the exception was not cleared by the Right Side and therefore has
     // not been handled. Zero means that the exception has been cleared. (Presumably, the debugger altered the
     // thread's context before clearing the exception, so continuing will give a different result.)
-    DWORD continueType = 0;
+	DWORD continueType = 0;
```

Comment:
Nit: tabs vs. spaces

Reply:
I fixed my editors tab settings and that shouldn't happen anymore.